documentation/docs/csv/index.md

```diff
@@ -227,14 +227,47 @@ implicit object MyRowDecoder extends CsvRowDecoder[MyRow, String] {
     s <- row.as[String]("s")
   } yield MyRow(i, j, s)
 }
-val decoded = withh.through(decodeRow[Fallible, String, MyRow])
+val decoded = withh.through(lowlevel.decodeRow[Fallible, String, MyRow])
 decoded.compile.toList
 ```
 
 Analogously, you can encode your data with a `CsvRowEncoder`. Make sure to not vary the headers based on the data itself as this can't be reflected in the CSV format.
 
 As you can see this can be quite tedious to implement. Lucky us, the `fs2-data-csv-generic` module comes to the rescue to avoid having to write the boilerplate. Please refer to [the module documentation][csv-generic-doc] for more details.
 
+### High-level API for standard use cases
```

Comment:
I would move this section up, before the low-level one, as it covers the standard use cases, let's see it first.

I think it's better for users to have it first, and then low-level saying: if the previous thing doesn't cover your needs, then have a look at low-level, ...

Reply:
I added an explanation of the API split and a quick-start using generic as that is what most people will be using. It has to forward to later sections for the type classes, but I believe it gives a good first impression of what the library offers and is easy to understand for people familiar with circe or similar. Does that fit your expectations?